modules/hcp-ibmz-dns.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-dns_{context}"]
+= DNS configuration for {hcp} on {ibm-z-title}
+
+The API server for the hosted cluster is exposed as a 'NodePort' service. A DNS entry must exist for the `api.<hosted_cluster_name>.<base_domain>` that points to the destination where the API server is reachable.
```

Comment:
Think this might should be:
```suggestion
The API server for the hosted cluster is exposed as a `NodePort` service type. A DNS entry must exist for the `api.<hosted_cluster_name>.<base_domain>` that points to the destination where the API server is reachable.
```
At least the markup.

Reply:
Good catch!